scripts/static_text/sitefooter/content_sitefooter.js

```diff
@@ -0,0 +1 @@
+$("#id_sitefooter").append("<br>Copyright 2019 - Neurobi Co.")
```

Comment:
(Review comment 5) I see what you did here! Good change since we god rid of the table in sitefooter. Is there any way we can dynamically infer the year? I wouldn't want to add much complexity to add this feature - maybe nothing more complex than a script tag import in the index.html file - if it gets hairier than that then forget I mentioned it! Good potential topic for the issues section